src/mame/bfm/bfm_bd1.cpp

```diff
@@ -319,21 +371,31 @@ int bfm_bd1_device::write_char(int data)
 				}
 				break;
 
-			case 0xC0:  // 0xC0 - 0xCF Set flash rate
-				m_flash_rate = data & 0x0F;
+			case 0xc0:  // 0xC0 - 0xCF Set flash rate
+				m_flash_rate = data & 0x0f;
+				if ( !m_flash_rate && m_flash )
+				{
+					m_flash_timer = 0;
+					m_flash = false;
+				}
+				m_flash_timer = 20;
```

Comment:
The original code had this fallthrough, I believe the intention is to turn the flag off, rather than set the timer to 0. However, the one section of datasheet that describes this behaviour fully is illegible, so this is all based off hardware videos. 20 cycles as a basis seems about right, but it's a guess.